extra/man/alacritty.5.scd

```diff
@@ -689,14 +689,73 @@ https://docs.rs/winit/\*/winit/keyboard/enum.Key.html
 
 	*action*
 
+		*ReceiveChar*
+			Allow receiving char input.
+		*None*
+			No action.
+		*Paste*
+			Paste contents of system clipboard.
+		*Copy*
+			Store current selection into clipboard.
+		*IncreaseFontSize*
+			Increase font size.
+		*DecreaseFontSize*
+			Decrease font size.
+		*ResetFontSize*
+			Reset font size to the config value.
+		*ScrollPageUp*
+			Scroll exactly one page up.
+		*ScrollPageDown*
+			Scroll exactly one page down.
+		*ScrollHalfPageUp*
+			Scroll half a page up.
+		*ScrollHalfPageDown*
+			Scroll half a page down.
+		*ScrollLineUp*
+			Scroll one line up.
+		*ScrollLineDown*
+			Scroll one line down.
+		*ScrollToTop*
+			Scroll all the way to the top.
+		*ScrollToBottom*
+			Scroll all the way to the bottom.
+		*ClearHistory*
+			Clear the display buffer(s) to remove history.
+		*Hide*
+			Hide the Alacritty window.
+		*Minimize*
+			Minimize the Alacritty window.
+		*Quit*
+			Quit Alacritty.
+		*ClearLogNotice*
+			Clear warning and error notices.
+		*SpawnNewInstance*
+			Spawn a new instance of Alacritty.
+		*CreateNewWindow*
+			Create a new Alacritty window.
+		*ToggleFullscreen*
+			Toggle fullscreen.
+		*ToggleMaximized*
+			Toggle maximized.
+		*ClearSelection*
+			Clear active selection.
+		*ToggleViMode*
+			Toggle vi mode.
+		*SearchForward*
+			Start a forward buffer search.
+		*SearchBackward*
+			Start a backward buffer search.
+
+		_Extra vi mode actions:_
```

Comment:
I wanted to emphasize that it's extra actions and not only actions. So you could still do other actions in it.

Reply:
Maybe rather than "extra vi mode actions" we should name it "Vi mode exclusive actions"?

That's how we've talked about this in the past.

Reply:
The thing is that you can bind them outside Vi mode due to pipelining we've added.

Reply:
But they're exclusive to the vi mode. They only work in vi mode.

Reply:
Maybe just `Vi mode actions` then?

Reply:
Sounds good to me.